rule.go

```diff
@@ -491,6 +491,15 @@ func (r *Rule) Contents() []*Content {
 	return cs
 }
 
+func (r *Rule) LastContent() *Content {
```

Comment:
can you add a test for this please?

Reply:
Just pushed, tried to keep your test style